docs/Use cases/Air_quality_simulators_use_case.md

```diff
@@ -0,0 +1,115 @@
+# Creating an air quality monitoring - Coiote DM and Azure IoT Hub integration use case
```

Comment:
The title is too long in my opinion, maybe just Air quality monitoring example?

Reply:
Went for "Air quality monitoring - use case". Is that ok?